packages/framework/src/Model/Product/Accessory/ProductAccessoryFacade.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
+
```

Comment:
I really do not mind it added. But maybe it is unwanted here?

Reply:
fixed